colossalai/device/device_mesh.py

```diff
@@ -18,9 +19,18 @@ class DeviceMesh:
             communication cost (default: None)
         mesh_beta (List[float], optional): coefficients used for computing
             communication cost (default: None)
+        init_process_group_during_init (bool, optional): initialize logical process group
```

Comment:
This variable name is kind of  tongue-twisting, I think `init_process_group` will be enough.

Reply:
hhhh，I will change it.